include/TS3.h

```diff
@@ -16,14 +16,14 @@ class TS3 : public TGRSIDetector {
 	public:
 
 		enum ES3Bits {
-			kPixelsSet 			= BIT(0),
-			kMultHit     	 	= BIT(1),
-			kBit2        		= BIT(2),
-			kBit3         		= BIT(3),
-			kBit4         		= BIT(4),
-			kBit5  				= BIT(5),
-			kBit6   				= BIT(6),
-			kBit7   				= BIT(7)
+			kPixelsSet	= BIT(0),
+			kMultHit	= BIT(1),//Should this be moved to a static?
+			kPreSector	= BIT(2),//Should this be moved to a static?
```

Comment:
For example, in tigress we have two bit holders, one is static 
		static TTransientBits<UShort_t> fgTigressBits;
Those two settings above kMultHit & kPreSector currently have to be changed to the users preference for the experiment inside the event loop every time new TS3 data is loaded.